statline_bq/cli.py

```diff
@@ -25,15 +32,21 @@ def upload_datasets():
     datasets_path = Path("./datasets.toml")
     config = get_config(config_path)
     datasets = get_datasets(datasets_path)
+    gcp_type = gcp_type.lower()
+    if gcp_type == "dev":
+        gcp_project = config.gcp.dev
```

Comment:
@galamit86 
it's good practice to use dicts for case switching in Python, see e.g. [this article](https://medium.com/python-in-plain-english/use-python-dictionary-as-a-switch-2b1f5450d46a)

```
config_envs = {'dev': config.gcp.dev, 'test': config.gcp.test, 'prod': config.gcp.prod}
gcp_project = config_envs.get(gcp_type)
```

Reply:
Nice, thanks for the tip!

Implemented, also [here](https://github.com/dataverbinders/statline-bq/blob/a83259c8bd8e71c5bf52620bd5850b103fa56212/statline_bq/utils.py#L378).